docs/Usage/CellMarkers.md

```diff
@@ -0,0 +1,73 @@
+
+### Cell Markers
+
+A cell marker is a special inline comment that defines the lines that begin each cell. The inline comment must start with the comment character registered in the language's grammar file.
+
+- Python: `#`
+- R: `#`
+- Julia: `#`
+- Javascript: `//`
+- C++: `//`
+- Stata: `//`
+- Matlab: `%`
+
+#### Code Cells
+
+After the comment character, you need to add one space plus any of the following markers on the same line:
+
+- `%%`
+- `<codecell>`
+- `In[]` (you can have any number of digits or spaces in the brackets)
+
+For example, the following text corresponds to three code cells in a Python file
+```py
+# %%
+print('Hello, world!')
+# <codecell>
+print('foo')
+# In[1]
+print('bar')
+```
+
+These are the same cell markers used by [**"Hydrogen: Run Cell"**](GettingStarted.md#hydrogen-run-cell).
+
+#### Markdown Cells
+
+You can also create cells with [Markdown](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)-formatted text. Just as for code cells, there are a range of allowed markers to delimit the start of a markdown cell:
```

Comment:
We don't use github markdown, but rather the "common markdown" specification.

Reply:
Replaced the link